Emby.Server.Implementations/Udp/UdpServer.cs

```diff
@@ -40,8 +40,8 @@ public UdpServer(ILogger logger, IServerApplicationHost appHost, IJsonSerializer
             _json = json;
             _socketFactory = socketFactory;
 
-            AddMessageResponder("who is EmbyServer?", true, RespondToV2Message);
-            AddMessageResponder("who is MediaBrowserServer_v2?", false, RespondToV2Message);
+            AddMessageResponder("who is JellyfinServer?", true, RespondToV2Message);
+            AddMessageResponder("who is Jellyfin?", true, RespondToV2Message);
```

Comment:
Is this one necessary?

Reply:
Don't know why I did that, removed it with c8bca53